### PR TITLE
Security.tl.xlf file

### PR DESCRIPTION
**Pull request by ghost**
| Q             | A
| ------------- | ---
| Branch?       | master 
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no 
| Tests pass?   | no
| Fixed tickets | none
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf [hunk 1]
```diff
@@ -3,63 +3,63 @@
     <file source-language="en" datatype="plaintext" original="file.ext">
         <body>
             <trans-unit id="1">
-                <source>An authentication exception occurred.</source>
+                <source>Isang pambihirang pagpaptunay ang nangyari</source>
```
Comment (xabbuh) [inline src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf]: the changes in this file need to be reverted

File: src/Symfony/Component/Validator/Resources/translations/validators.af.xlf [hunk 2]
```diff
@@ -3,223 +3,223 @@
     <file source-language="en" datatype="plaintext" original="file.ext">
         <body>
             <trans-unit id="1">
-                <source>This value should be false.</source>
+                <source>Ang halaga nito ay dapat na huwad.</source>
```
Comment (xabbuh) [inline src/Symfony/Component/Validator/Resources/translations/validators.af.xlf]: same here

**Comment by xabbuh** [comment]
you will also need to rename the `.tl.xlf` file to `security.tl.xlf`

**Comment by ergiegonzaga** [comment]
Hi sir, how to submit pull request that I will not include previous submitted files?

**Comment by stof** [comment]
@ergiegonzaga create a branch starting from the **symfony** master branch, not from the master branch of your fork (as your own master branch is the source of the previous PR)

**Comment by nicolas-grekas** [comment]
Closing in favor of #26130

**Pull request closed without merging**